samples/net/echo_client/prj_qemu_x86.conf

```diff
@@ -33,5 +33,5 @@ CONFIG_NET_APP_SETTINGS=y
 # addresses (so that peer address is ending to 2).
 CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::2"
 CONFIG_NET_APP_PEER_IPV6_ADDR="2001:db8::1"
-CONFIG_NET_APP_MY_IPV4_ADDR="192.0.2.2"
-CONFIG_NET_APP_PEER_IPV4_ADDR="192.0.2.1"
+CONFIG_NET_APP_MY_IPV4_ADDR="192.0.2.1"
```

Comment:
There was a reason for this swap for qemu_x86 and qemu_cortex_m3 (although this is not documented in arm conf file). See the comment few lines above, we have the addresses swapped so that we can simply test the qemu<->qemu communication.

Reply:
@jukkar Ohh, sorry. I was using cortex_m3.conf I thought it was wrong then I just greped to see where is the similar situation without reading the rest of .conf file. I will change this patch to just improve documentation of prj_qemu_cortex_m3.conf